infra/gcp/namespaces/namespace-user-role.yml

```diff
@@ -12,7 +12,7 @@ rules:
     resources: ["horizontalpodautoscalers"]
     verbs: ["*"]
   - apiGroups: ["apps"]
-    resources: ["deployments"]
+    resources: ["deployments","replicasets","statefulsets"]
```

Comment:
I'll start the topic at our todays call with @thockin, but as a nit could you add spaces after the commas maybe? :-)

Reply:
It's done.